doc/source/ray-more-libs/dask-on-ray.rst

```diff
@@ -272,14 +271,18 @@ execution time exceeds some user-defined threshold:
 
 This callback API is currently unstable and subject to change.
 
+API
+---
+
 .. autosummary::
    :nosignatures:
    :toctree: doc/
 
-   ray.util.dask.callbacks.RayDaskCallback
-   ray.util.dask.callbacks.RayDaskCallback._ray_presubmit
-   ray.util.dask.callbacks.RayDaskCallback._ray_postsubmit
-   ray.util.dask.callbacks.RayDaskCallback._ray_pretask
-   ray.util.dask.callbacks.RayDaskCallback._ray_posttask
-   ray.util.dask.callbacks.RayDaskCallback._ray_postsubmit_all
-   ray.util.dask.callbacks.RayDaskCallback._ray_finish
+   ray.util.dask.RayDaskCallback
```

Comment:
this class is inside the `callbacks.py` module; why do we remove the `callbacks` part?

Reply:
This has to do with the way that `sphinx.ext.autosummary` resolves imports to the most base module where it is exported. Here, `RayDaskCallback` is explicitly exported as part of `ray.util.dask.__init__.py`: https://github.com/ray-project/ray/blob/18acf026d23c6520523e12a31d251773fee00c54/python/ray/util/dask/__init__.py#L57. The link won't resolve otherwise.